dev/com.ibm.ws.transport.http/src/com/ibm/ws/genericbnf/internal/BNFHeadersImpl.java

```diff
@@ -866,9 +876,13 @@ public HeaderField getHeader(String header) {
         if (null == header) {
             throw new IllegalArgumentException("Null input provided");
         }
-        HeaderElement elem = findHeader(findKey(header));
-        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
-            Tr.debug(tc, "getHeader(s): " + header + " " + elem);
+        HeaderElement elem = null;
+        HeaderKeys key = headerKeysByName.get(header);
+        if (key != null) {
+            elem = findHeader(key);
+            if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                Tr.debug(tc, "getHeader(s): " + header + " " + elem);
+            }
         }
```

Comment:
Similar lose of debug statement:

```suggestion
        HeaderElement elem = null;
        HeaderKeys key = headerKeysByName.get(header);
        if (key != null) {
            elem = findHeader(key);
        }
        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
            Tr.debug(tc, "getHeader(s): " + header + " " + elem);
        }
```